pkg/ddc/thin/transform_fuse.go

```diff
@@ -180,16 +169,6 @@ func (t *ThinEngine) parseFuseOptions(runtime *datav1alpha1.ThinRuntime, profile
 		options[k] = v
 	}
 
-	mountOptions, err := t.genUFSMountOptions(dataset.Spec.Mounts[0], dataset.Spec.SharedOptions, dataset.Spec.SharedEncryptOptions)
-	if err != nil {
-		t.Log.Info("Error:", "err", err)
-		return "", err
-	}
-
-	for k, v := range mountOptions {
-		options[k] = v
-	}
-
```

Comment:
For reviewers: This is to remove mount options extracted from dataset.spec.mounts like `options` and `encryptOptions`. It is supposed that FUSE should get required options from  `config.json` and `/etc/fluid/secrets` instead of env variable.